server/controllers/appointment.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler =  require('express-async-handler');
+const validatorResults = require('express-validator');
+const Appointment = require('../models/Appointment');
+const Meeting = require('../models/Meeting');
+const moment = require('moment-timezone');
+//CREATE /appointment -> Create an appointment
+//meeting_id, name, email, time (datetime), timezone
+//GET /appointments -> list of appointments for logged in user
```

Comment:
We don't need this here, below has some documentation on the controller method.

Reply:
Removed.